inspec.gemspec

```diff
@@ -38,4 +38,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'sslshake', '~> 1'
   spec.add_dependency 'parallel', '~> 1.9'
   spec.add_dependency 'rspec_junit_formatter', '~> 0.2.3'
+  spec.add_dependency 'requests', '~> 1.0.0'
```

Comment:
@chris-rock or I just find http://www.rubydoc.info/gems/http/HTTP/Request https://github.com/httprb/http who is:
* quite maintained :)
* some more work to use it "as it" (basic auth etc) :|
* has some dependencies :(

It's up to you :)